src/server/scripts/EasternKingdoms/OperationGnomeregan/operation_gnomeregan.h

```diff
@@ -0,0 +1,503 @@
+    {-5081.82f, 451.51f, 410.33f, 2.46f},
+    {-5152.82f, 455.32f, 392.82f, 2.49f},
+    {-5142.35f, 468.65f, 392.38f, 2.33f},
+    {-5191.10f, 574.74f, 400.96f, 1.51f},
+    {-5177.70f, 574.41f, 400.40f, 1.51f},
+    {-4975.04f, 736.62f, 256.91f, 3.04f},
+    {-4976.25f, 723.63f, 256.91f, 3.04f},
+    {-5103.44f, 724.77f, 257.77f, 6.24f},
+    {-5109.18f, 725.26f, 255.98f, 6.24f},
+    {-5104.79f, 717.42f, 257.53f, 6.24f},
+    {-5109.95f, 717.81f, 255.92f, 6.24f}
+};
+
+const Position RLSpawn[4] =
+{
+    {-5350.35f, 574.63f, 387.12f, 4.7f},
+    {-5331.62f, 545.17f, 385.22f, 5.0f},
+    {-5324.26f, 588.66f, 389.01f, 3.7f},
+    {-5294.58f, 571.73f, 386.93f, 5.3f}
+};
+
+const Position CannonSpawn[6] =
+{
+    {-5140.57f, 428.39f, 395.89f, 2.1f},
+    {-5109.37f, 426.51f, 399.66f, 2.4f},
+    {-5109.00f, 481.10f, 398.45f, 3.9f},
+    {-5071.21f, 457.83f, 410.94f, 3.0f},
+    {-5087.23f, 434.92f, 411.78f, 1.9f},
+    {-5092.15f, 450.84f, 406.63f, 1.4f}
+};
+
+const Position BattleSuitDriverSpawn[3] =
+{
+    {-5158.31f, 470.63f, 390.47f, 5.0f},
+    {-5156.70f, 473.46f, 390.56f, 5.0f},
+    {-5154.23f, 475.71f, 390.82f, 5.0f}
+};
+
+const Position BattleSuitSpawn[6] =
+{
+    {-5073.81f, 481.83f, 401.48f, 5.0f},
+    {-5069.55f, 485.22f, 401.49f, 4.8f},
+    {-5064.04f, 486.30f, 401.48f, 4.8f},
+    {-5086.02f, 713.15f, 260.56f, 6.3f},
+    {-5085.04f, 725.92f, 260.56f, 6.3f},
+    {-5085.31f, 477.48f, 401.95f, 3.9f}
+};
+
+const Position TankSpawn[5] =
+{
+    {-5402.421387f, 543.972229f, 387.243896f, 6.1f},
+    {-5404.228516f, 530.777527f, 387.151581f, 0.2f},
+    {-5394.222656f, 519.042908f, 386.308929f, 1.0f},
+    {-5280.954102f, 559.265808f, 385.679932f, 5.0f},
+    {-5263.007813f, 572.262085f, 388.673157f, 5.0f}
+};
+
+const Position BragBotSpawn[2] =
+{
+    {-5183.73f, 599.90f, 408.96f, 4.6f},
+    {-4933.62f, 717.18f, 261.65f, 1.6f}
+};
+
+const uint32 Worldstates[15] =
+{
+    4981,
+    5027,
+    5040,
+    5018,
+    5039,
+    5044,
+    5006,
+    5018,
+    5043,
+    5019,
+    5045,
+    5021,
+    5011,
+    5037,
+    5038
+};
+
+const Position BomberSpawn              = {-5314.53f, 564.82f, 391.43f, 5.7f};
+const Position RadControlSpawn          = {-5072.80f, 441.48f, 410.97f, 2.6f};
+
+const Position TroggSpawn               = {-5181.74f, 631.21f, 398.54f, 4.7f};
+
+const Position ExplosionBunnySpawn      = {-5183.24f, 608.97f, 410.89f, 4.7f};
+
+// Texts in Prepping the Speech
+enum MEK_T
+{
+    MEK1_0_0 = 0,
+    MEK1_0_1,
+    MEK1_0_2,
+    MEK1_0_3,
+    MEK1_0_4,
+    MEK1_0_5,
+    MEK1_0_6,
+    MEK1_0_7,
+
+    MEK1_REPLY = 0,
+};
+
+enum STEAM_Texts
+{
+    STEAM_0 = 0,
+    STEAM_1,
+    STEAM_2,
+    STEAM_3,
+    STEAM_4,
+    STEAM_5,
+    STEAM_6,
+    STEAM_7,
+    STEAM_8,
+    STEAM_9,
+    STEAM_10,
+    STEAM_11,
+    STEAM_12,
+    STEAM_13,
+    STEAM_14,
+    STEAM_15,
+    STEAM_16,
+    STEAM_17,
+    STEAM_18,
+};
+
+#define MEK_1_1                 "Citizens and Friends of Gnomeregan!"
```

Comment:
all of  this should be written in proper creature_text/gossip/whatever entries and not hardcoded